pkg/pluginhelper/http/server_test.go

```diff
@@ -81,8 +81,14 @@ var _ = Describe("BuildTLSConfig", func() {
 		tlsConfig, err := server.buildTLSConfig(ctx)
 		Expect(err).ToNot(HaveOccurred())
 		Expect(tlsConfig).ToNot(BeNil())
-		Expect(tlsConfig.Certificates).To(HaveLen(1))
+		Expect(tlsConfig.GetCertificate).ToNot(BeNil())
 		Expect(tlsConfig.ClientCAs.Subjects()).ToNot(BeEmpty()) //nolint: staticcheck
 		Expect(tlsConfig.MinVersion).To(Equal(uint16(tls.VersionTLS13)))
+
+		cert, err := tlsConfig.GetCertificate(&tls.ClientHelloInfo{
```

Comment:
Not entirely sure on this part.

Reply:
Since the first argument of the function is not used, you can pass nil instead.